builder/exector/export_app.go

```diff
@@ -640,7 +640,11 @@ func (i *ExportApp) buildDockerComposeYaml() error {
 		return err
 	}
 
-	err = ioutil.WriteFile(fmt.Sprintf("%s/docker-compose.yaml", i.SourceDir), content, 0644)
+	// sed -i 's/""//g' docker-compose.yaml
+	contentStr := strings.ReplaceAll(string(content), "\"\"", "")
```

Comment:
You should directly prevent volume name= ""

Reply:
I have no idea to do this after googling, do you know how to do it?

Reply:
fixed